packages/expo-passkeys/src/index.ts

```diff
@@ -0,0 +1,191 @@
+      ),
+    default: null,
+  });
+
+  if (!createPasskeyModule) {
+    throw new Error('Platform not supported');
+  }
+
+  try {
+    const response = await createPasskeyModule();
+    return {
+      publicKeyCredential: makeSerializedCreateResponse(typeof response === 'string' ? JSON.parse(response) : response),
+      error: null,
+    };
+  } catch (error) {
+    return {
+      publicKeyCredential: null,
+      error: mapNativeErrorToClerkWebAuthnErrorCode(error.code, error.message, 'create'),
+    };
+  }
+}
+
+const makeSerializedGetResponse = (
+  publicKeyCredential: AuthenticationResponseJSON,
+): PublicKeyCredentialWithAuthenticatorAssertionResponse => {
+  return {
+    type: publicKeyCredential.type,
+    id: publicKeyCredential.id,
+    rawId: base64urlToArrayBuffer(publicKeyCredential.rawId),
+    authenticatorAttachment: publicKeyCredential?.authenticatorAttachment || null,
+    response: {
+      clientDataJSON: base64urlToArrayBuffer(publicKeyCredential.response.clientDataJSON),
+      authenticatorData: base64urlToArrayBuffer(publicKeyCredential.response.authenticatorData),
+      signature: base64urlToArrayBuffer(publicKeyCredential.response.signature),
+      userHandle: publicKeyCredential?.response.userHandle
+        ? base64urlToArrayBuffer(publicKeyCredential?.response.userHandle)
+        : null,
+    },
+    toJSON: () => publicKeyCredential,
+  };
+};
+
+export async function get({
+  publicKeyOptions,
+}: {
+  publicKeyOptions: PublicKeyCredentialRequestOptionsWithoutExtensions;
+}): Promise<CredentialReturn<PublicKeyCredentialWithAuthenticatorAssertionResponse>> {
+  if (!publicKeyOptions) {
+    throw new Error('publicKeyCredential has not been provided');
+  }
+
+  const serializedPublicCredential: SerializedPublicKeyCredentialRequestOptions = {
+    ...publicKeyOptions,
+    challenge: arrayBufferToBase64Url(publicKeyOptions.challenge),
+  };
+
+  const getPasskeyModule = Platform.select({
+    android: async () => ClerkExpoPasskeys.get(JSON.stringify(serializedPublicCredential)),
+    ios: async () => ClerkExpoPasskeys.get(serializedPublicCredential.challenge, serializedPublicCredential.rpId),
+    default: null,
+  });
+
+  if (!getPasskeyModule) {
+    return {
+      publicKeyCredential: null,
+      error: new ClerkWebAuthnError('Platform is not supported', { code: 'passkey_not_supported' }),
+    };
+  }
+
+  try {
+    const response = await getPasskeyModule();
+    return {
+      publicKeyCredential: makeSerializedGetResponse(typeof response === 'string' ? JSON.parse(response) : response),
+      error: null,
+    };
+  } catch (error) {
+    return {
+      publicKeyCredential: null,
+      error: mapNativeErrorToClerkWebAuthnErrorCode(error.code, error.message, 'get'),
+    };
+  }
+}
+
+export function isSupported() {
+  if (Platform.OS === 'android') {
+    return Platform.Version >= 28;
+  }
+
+  if (Platform.OS === 'ios') {
+    return parseInt(Platform.Version, 10) > 15;
+  }
```

Comment:
Can we extract these version numbers to be variables and also have a comment mentioning which andriod/ios version they relate to? For example, the `28` corresponds to Android 9.

ref: https://apilevels.com

Reply:
I think we can skip the comment by introducing the variables like this.

```
const ANDROID_9 = 28;
const IOS_15 = 15;
```

WDYT?

Reply:
LGTM! 💯